### ISSUE TITLE
Resize on Orientation Change

### ISSUE THREAD

**Issue by simerlec** [highcharts/highcharts-ios#20]
Hello,

maybe this is a trivial issue but I'm fairly new to iOS development so bear with me :)

Currently I have 2 charts in a scrollview below one another - due to the fact that they are generated dynamically I also created the layout constraints in code. Everything works as expected so far except that the charts don't reflow after an orientation change from portrait to landscape and vice versa.

```swift
let chartView2 = HIGChartView(frame: CGRect(x: 0, y: 0, width: UIScreen.main.bounds.width, height: 400))
chartView2.translatesAutoresizingMaskIntoConstraints = false  //needed for programmatic constraints
let options2 = HIOptions()
let chart2 = HIChart()
let title2 = HITitle()
let series2 = HISeries()
title2.text = "Demo"
chart2.type = "column"
options2.chart = chart2
series2.data = data
options2.series = [series2]
chartView2.options = options2

scrollView.addSubview(chartView2)
chartView2.topAnchor.constraint(equalTo: pieChart.bottomAnchor).isActive = true
chartView2.bottomAnchor.constraint(equalTo: scrollView.bottomAnchor).isActive = true
chartView2.heightAnchor.constraint(equalToConstant: 400).isActive = true
chartView2.widthAnchor.constraint(equalToConstant: UIScreen.main.bounds.width).isActive = true
```

Currently I'm using this self-made workaround which works fine so far I guess...

```swift
NotificationCenter.default.addObserver(self, selector: #selector(ViewController.rotated), name: NSNotification.Name.UIDeviceOrientationDidChange, object: nil)

func rotated() {
    if UIDevice.current.orientation.isLandscape {
         chartView2.options.chart.width = UIScreen.main.bounds.width as NSNumber
         chartView2.reload()
    } else {
         chartView2.options.chart.width = UIScreen.main.bounds.width as NSNumber
         chartView2.reload()
    }
} 
```

...but is there a better/easier/right way?

kr and keep up the good work

**Comment by mll** [comment]
This is a bug that will be fixed in the next release, fix is coming soon.

**Comment by simerlec** [comment]
Thanks for the fast response!

**Comment by mll** [comment]
This is fixed in 1.1.0
